modules/apps/collaboration/message-boards/message-boards-service/src/main/java/com/liferay/message/boards/internal/pop/MessageListenerImpl.java

```
@@ -249,9 +249,10 @@ public void deliver(String from, String recipient, Message message)
 				for (ObjectValuePair<String, InputStream> inputStreamOVP :
 						inputStreamOVPs) {
 
-					InputStream inputStream = inputStreamOVP.getValue();
-
-					StreamUtil.cleanUp(inputStream);
+					try (InputStream inputStream = inputStreamOVP.getValue()) {
+					}
+					catch (IOException ioe) {
```

Why does this one have the empty catch but the one in MBMessageStagedModelDataHandler.java does not?